lib/elixir/test/elixir/protocol_test.exs

```diff
@@ -387,7 +394,9 @@ defmodule Protocol.ConsolidationTest do
 
   test "consolidation updates attributes" do
     assert Sample.__protocol__(:consolidated?)
+    assert Sample.__protocol__(:impls) == [ImplStruct]
```

Comment:
And I think we should return the path for the whole implementation and not the "type".